icons/eclipse.svg

```diff
@@ -0,0 +1 @@
5469 0.105469 C 6.726562 0.417969 3.480469 2.566406 1.597656 5.894531 C 0.71875 7.449219 0.175781 9.269531 0.046875 11.125 C 0.0078125 11.640625 0.0273438 12.863281 0.0820312 13.324219 C 0.386719 15.996094 1.429688 18.363281 3.140625 20.261719 C 5.09375 22.425781 7.707031 23.714844 10.546875 23.910156 C 10.964844 23.941406 11.585938 23.949219 11.5625 23.921875 C 11.554688 23.914062 11.40625 23.878906 11.226562 23.847656 C 9.558594 23.519531 7.851562 22.722656 6.46875 21.625 C 5.5625 20.90625 4.691406 19.972656 4.027344 19 C 2.730469 17.105469 1.980469 14.8125 1.902344 12.507812 C 1.847656 10.992188 2.027344 9.636719 2.460938 8.25 C 3.242188 5.738281 4.820312 3.542969 6.917969 2.046875 C 8.21875 1.113281 9.570312 0.519531 11.136719 0.179688 L 11.578125 0.0820312 L 11.089844 0.0820312 C 10.816406 0.0859375 10.488281 0.09375 10.355469 0.105469 Z M 10.355469 0.105469 "/><path d="M 12.558594 0.589844 C 10.628906 0.707031 8.796875 1.351562 7.296875 2.449219 C 5.429688 3.8125 4.003906 5.679688 3.234375 7.773438 C 3.089844 8.167969 2.953125 8.589844 2.953125 8.648438 C 2.953125 8.660156 6.765625 8.671875 13.222656 8.671875 C 21.652344 8.671875 23.496094 8.664062 23.496094 8.636719 C 23.496094 8.589844 23.34375 8.113281 23.214844 7.761719 C 23.0625 7.34375 22.640625 6.457031 22.398438 6.046875 C 22 5.367188 21.492188 4.679688 20.929688 4.0625 C 19.285156 2.25 17.351562 1.140625 15.132812 0.742188 C 14.351562 0.601562 13.339844 0.542969 12.558594 0.589844 Z M 12.558594 0.589844 "/><path d="M 13.195312 9.992188 L 2.664062 10 L 2.652344 10.058594 C 2.617188 10.21875 2.519531 11.136719 2.519531 11.296875 C 2.519531 11.324219 4.296875 11.328125 13.222656 11.328125 L 23.929688 11.328125 L 23.929688 11.253906 C 23.929688 11.152344 23.859375 10.464844 23.824219 10.203125 C 23.804688 10.042969 23.785156 9.988281 23.761719 9.988281 C 23.742188 9.988281 18.988281 9.988281 13.195312 9.992188 Z M 13.195312 9.992188 "/><path d="M 2.53125 12.648438 C 2.511719 12.6875 2.554688 13.203125 2.605469 13.617188 C 2.625 13.773438 2.640625 13.914062 2.640625 13.929688 C 2.640625 13.957031 4.769531 13.964844 13.222656 13.964844 L 23.804688 13.964844 L 23.828125 13.777344 C 23.871094 13.453125 23.929688 12.8125 23.921875 12.714844 L 23.914062 12.621094 L 13.230469 12.613281 C 3.898438 12.609375 2.542969 12.613281 2.53125 12.648438 Z M 2.53125 12.648438 "/><path d="M 2.953125 15.355469 C 2.953125 15.375 3.007812 15.5625 3.074219 15.773438 C 3.519531 17.148438 4.191406 18.359375 5.136719 19.5 C 5.417969 19.839844 6.078125 20.53125 6.359375 20.78125 C 7.921875 22.175781 9.59375 22.988281 11.519531 23.300781 C 12.121094 23.394531 12.402344 23.414062 13.261719 23.410156 C 14.167969 23.410156 14.554688 23.378906 15.277344 23.234375 C 17.378906 22.820312 19.175781 21.789062 20.785156 20.078125 C 22.046875 18.734375 22.882812 17.328125 23.402344 15.683594 L 23.519531 15.320312 L 13.234375 15.320312 C 4.386719 15.320312 2.953125 15.324219 2.953125 15.355469 Z M 2.953125 15.355469 "/></svg>
```

Comment:
:+1: Nice work on the SVG! Could you give it a run through [SVGOMG](https://jakearchibald.github.io/svgomg/) to see if you can lower the precision a bit without introducing artifacts? I usually find a precision of 3 to be the sweet spot. Also remember to update the `<title>` when you update it in the JSON :)